hyperspy/io_plugins/emd.py

```diff
@@ -128,6 +130,9 @@ class EMD(object):
 
     def __init__(self, signals=None, user=None,
                  microscope=None, sample=None, comments=None):
+        warnings.warns("Direct instantiation of the EMD class is deprecated "
+                       "and will be removed in HyperSpy v2.0. Please use the "
+                       "`hs.load` function instead.")
```

Comment:
Should this raise a `VisibleDeprecationWarning` like usual?

Reply:
Done!